src/main/java/net/pms/PMS.java

```diff
@@ -366,7 +355,11 @@ private void displayBanner() throws IOException {
 		 * usually done by the installer
 		 */
 		File dDir = new File(configuration.getDataDir());
-		dDir.mkdirs();
+		if (!dDir.exists()) {
```

Comment:
@Nadahar I assume there's a reason for the if statements being on two separate lines instead of one statement, what is it?